### PR TITLE
Add Colin

### PR DESCRIPTION
**Pull request by ColCarroll**
image

### CODE REVIEW

File: _data/twenty_17/teams.yml [hunk 1]
```diff
@@ -104,14 +104,14 @@ Sid was born in New Delhi and did a large share of his growing up in Melbourne.
...
-  topic:
-  description:
+  twitter: colindcarroll
+  github: ColCarroll
+  web: https://colindcarroll.com/
+  img: colin.png
+  thumbnail: colin.png
+  alt: Picture of Colin
+  topic: Colin is a contributor to PyMC3 and ArviZ. He is enthusiastic about open source science, data visualization, and his small dog Pete.
+  description: Colin is a software engineer at Google Research Cambridge. He is interested in statistical computing and visualization, particularly as related to Bayesian methods. He is heavily involved in open source: a core contributor to PyMC3, a Python library for Bayesian modelling and inference, as well as ArviZ, a Bayesian visualization and diagnostic library. He received his PhD in mathematics from Rice University, where he researched geometric measure theory.
```
Comment (OriolAbril) [inline _data/twenty_17/teams.yml:114]: the ":" are not allowed as is. the whole description should be surrounded by quotation marks or apostrophes

Comment (OriolAbril) [reply _data/twenty_17/teams.yml:114]: cc @canyon289 @ColCarroll I figured that would be easier than opening an issue explaining what is happening

Comment (canyon289) [reply _data/twenty_17/teams.yml:114]: Yep Ill fix just got busy

**Comment by ColCarroll** [comment]
#33 should fix

**Pull request merged**